rai/node/node.cpp

```diff
@@ -515,13 +515,15 @@ class network_message_visitor : public rai::message_visitor
 		}
 		node.stats.inc (rai::stat::type::message, rai::stat::detail::confirm_req, rai::stat::dir::in);
 		node.peers.contacted (sender, message_a.header.version_using);
-		node.process_active (message_a.block);
-		node.active.publish (message_a.block);
```

Comment:
This was added by vote-by-hash (#1025), probably to republish a block that's being voted on without the contents of the block.  I agree it seems un-needed here, atleast implicitly.  I will double-check with @PlasmaPower to find out of there's some other intention.

Reply:
Yeah I agree that if we're not processing the block, we don't need to call .publish